core/runtime/module.go

```diff
@@ -0,0 +1,54 @@
+package runtime
+
+import (
+	// Register automaxprocs to automaticly set GOMAXPROCS in container environments
+	"fmt"
+	"sync"
+
+	"flamingo.me/dingo"
+	"flamingo.me/flamingo/v3/framework/flamingo"
+	"go.uber.org/automaxprocs/maxprocs"
+)
+
+type (
+	// Module for core.runtime
+	Module struct {
+		logger flamingo.Logger
+	}
+)
+
+var once = sync.Once{}
+
+// Inject Module dependencies
+func (m *Module) Inject(logger flamingo.Logger) *Module {
+	m.logger = logger
+	return m
+}
+
+// Configure runtime dependency injection
+func (m *Module) Configure(injector *dingo.Injector) {
+	once.Do(func() {
+		_, err := maxprocs.Set(maxprocs.Logger(
+			func(logMessage string, args ...interface{}) {
+				m.logger.
+					WithFields(
```

Comment:
I think this is more readable:
WithField(flamingo.LogKeyModule,"core.runtime").WithField(flamingo.LogKeyCategory,"maxprocs")

Reply:
ok. i understand, but it writing like this creates a realy big line-length wich is not nice for readability to. i changed it so another variant which should increase readability and removes a duplication. hope that's ok..

Reply:
You can add a line break after the first method call